docs/en/operation/Metrics-System.md

```diff
@@ -67,6 +67,26 @@ $ curl <WORKER_HOSTNAME>:<WORKER_WEB_PORT>/metrics/json/
 $ curl 127.0.0.1:19999/metrics/json/
 # Get the local worker metrics with its default web port 30000
 $ curl 127.0.0.1:30000/metrics/json/
+``` 
+
+Send an HTTP request to `/metrics/json/` to fuse web port to get fuse metrics. Check out the [Fuse metrics doc]({{ '/en/api/POSIX-API.html' | relativize_url }}#fuse-metrics) for how to enable fuse metrics.
+
+```console
+# Get the metrics in JSON format from an active Alluxio FUSE process.
+$ curl <FUSE_HOSTNAME>:<FUSE_WEB_PORT>/metrics/json
+
+# For example, get the metrics from fuse process running locally with default web port
+$ curl 127.0.0.1:49999/metrics/json/
+``` 
+
+### CSV Sink Setup
```

Comment:
@jja725 is this section needed?
looks like it's not the full CSV sink setup
isn't it in the doc already?

Reply:
might be some problem when merging master. Thanks for finding this and Fixed